react/features/base/react/components/native/JitsiSectionList.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    SafeAreaView,
+    SectionList
+} from 'react-native';
+
+
+/**
+ * Implements a React/Web {@link Component} for displaying text similar to React
+ * Native's {@code Text} in order to faciliate cross-platform source code.
+ *
+ * @extends Component
+ */
+export default class JitsiSectionList extends Component {
```

Comment:
Maybe we can name it SectionList. WDYT?

Reply:
The reason I didnt name it `SectionList` is because it has an element because we are importing 
```
import {
    SafeAreaView,
    SectionList
} from 'react-native';
```
I could try changing the name of the react-native `SectionList` when I import it and then change the file name to SectionList